google/cloud/pubsub/integration_tests/subscription_lifecycle.cc

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 Google LLC
```

Comment:
Would we not typically name this file with an `_test` suffix?

Reply:
Renamed